AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -85,7 +85,7 @@ public EventManagerInviteController(IMediator mediator, UserManager<ApplicationU
 
                 invite.EventId = eventId;
                 var user = await _userManager.GetUserAsync(User);
-                await _mediator.SendAsync(new CreateEventManagerInviteCommand { Invite = invite, UserId = userToInvite?.Id });
+                await _mediator.SendAsync(new CreateEventManagerInviteCommand { Invite = invite, UserId = user.Id });
                 // TODO: Send invite
```

Comment:
please remove the //TODO comment here. The dispatching of the email will not be done through the controller.